File: Ex11/localstiff1D.jl
# Sample solution DP
function localstiff1D(Fdet::Float64, Finv::Float64)::Array{Float64,2}
    G = Finv;
    sloc = zeros(2, 2);
    sloc[1,1] =   G' * G * Fdet;
    sloc[1,2] = - G' * G * Fdet;
    sloc[2,1] = sloc[1,2];
    sloc[2,2] = sloc[1,1];
    return sloc;
end

File: Ex11/localstiff1D.m
% Sample solution DP
function sloc=localstiff1D(Fdet,Finv)
G = Finv;
sloc(1,1) =   G' * G * Fdet;
sloc(1,2) = - G' * G * Fdet;
sloc(2,1) = sloc(1,2);
sloc(2,2) = sloc(1,1);
end